stdlib/cProfile.pyi

```diff
@@ -5,6 +5,8 @@ from types import CodeType
 from typing import Any, TypeVar
 from typing_extensions import ParamSpec, Self, TypeAlias
 
+import _lsprof
```

Comment:
Could you add `"_lsprof"` to the table here, so that ruff puts the import in the right place?

https://github.com/python/typeshed/blob/3171cb44aa9d7c7ada31ef78720b42ea28acd24a/pyproject.toml#L50-L93